### DISCUSSION TITLE
Will Valkey support Redis 8's JSON datatype / commands?

### DISCUSSION THREAD

**Discussion by binaryfire** [valkey-io/valkey#1306] [Q&A]
Hi

According to the [docs](https://valkey.io/commands/) Valkey doesn't support [Redis 8's new JSON commands](https://redis.io/docs/latest/commands/?name=json). Will this datatype be added in a future release?

**Comment by madolson** [comment]
We have a plan for JSON documented here: https://github.com/valkey-io/valkey-rfc/blob/main/ValkeyJSON.md. You can follow the conversation there.
